### ISSUE TITLE
end mobs bug

### ISSUE THREAD

**Issue by celophys** [Alex-the-666/AlexsMobs#435]
endergrade and enderiophage are extremly rare, also after changing the spawn value in the config
with last version of alexmobs and citadel

**Comment by pm065** [comment]
Yeah, I haven't been seeing any of them. Oddly, Spectres don't seem affected.
